packages/hooks/src/useCurrentRegion.js

```diff
@@ -11,11 +11,5 @@ async function fetchRegion() {
 }
 
 export default function useCurrentRegion(options) {
-  const { data = null, isFetching, error } = useQuery(
-    ['region'],
-    fetchRegion,
-    options
-  );
-
-  return [data, isFetching, error];
+  return useQuery(['region'], fetchRegion, options);
```

Comment:
in the `fetchRegion` function above we want to make sure it's safe to access the response data since it may be undefined, I think we should update this and other functions like it to something like `response?.data?.regions?.[0]?.id`